tests/test_request_handlers.py

```diff
@@ -55,19 +55,14 @@ def get(h_self):
             (r'/without-mixin', WithoutMixinRequestHandler),
         )
 
-        self._factory = make_session_factory(postgres_url)
-        self._application = Application(
-            handlers, session_factory=self._factory
-        )
+        db.configure(uri=mysql_url)
+
+        self._application = Application(handlers, db=db)
 
     def setUp(self, *args, **kwargs):
         super(RequestHandlersTestCase, self).setUp(*args, **kwargs)
 
-        Base.metadata.create_all(self._factory.engine)
```

Comment:
Actually now that the `create_all` function is removed, the `setUp` function doesn't need to be overridden at all.